neu-self-image-experiments/sie-firebase
# Experiment task full screen

Used the `Modal` component to have the jsPsych selection task take up the whole screen. `ScreenTakeover` was giving me issues with styling.

- There's a button that opens up the `Modal` and shows the selection task on Step 5
- `Next` button to go to the next step is disabled until the experiment is complete

Some ui/ux issues I found that I could use some input on, we can put these into a new ticket but wanna get some ideas first:

1. Once the modal is open, the focus isn't on the selection div so we need to fix that again. You have to click on the "task" for key input events to work.
2. After the experiment ends, you need to press a key to fully complete and got to the next step. Also once you complete, the data gets input onto the screen?
3. Is you close the modal before the selection task ends, it still runs in the background, does jsPsych have a way to pause experiments?

To test
- npm start
- [http://localhost:3000/9iG2wajKXkjLXiLQfwgj/Bo0Hka0QT5O2Q026qKpFbAYHOW23/](http://localhost:3000/9iG2wajKXkjLXiLQfwgj/Bo0Hka0QT5O2Q026qKpFbAYHOW23/)
- go to step 5
- notice that next button is disabled
- press button to open modal
- run the experiment all the way through
- check if the button is enabled
